src/main/java/org/jabref/gui/preftabs/ExportSortingPrefsTab.java

```diff
@@ -1,81 +1,74 @@
-        builder.leadingColumnOffset(1);
-
         // EXPORT SORT ORDER
         // create Components
-        exportInOriginalOrder = new JRadioButton(Localization.lang("Export entries in their original order"));
-        exportInTableOrder = new JRadioButton(Localization.lang("Export in current table sort order"));
-        exportInSpecifiedOrder = new JRadioButton(Localization.lang("Export entries ordered as specified"));
-
-        ButtonGroup buttonGroup = new ButtonGroup();
-        buttonGroup.add(exportInOriginalOrder);
-        buttonGroup.add(exportInTableOrder);
-        buttonGroup.add(exportInSpecifiedOrder);
+        exportInOriginalOrder = new RadioButton(Localization.lang("Export entries in their original order"));
+        exportInOriginalOrder.setFont(FontSize.smallFont);
+        exportInTableOrder = new RadioButton(Localization.lang("Export in current table sort order"));
+        exportInTableOrder.setFont(FontSize.smallFont);
+        exportInSpecifiedOrder = new RadioButton(Localization.lang("Export entries ordered as specified"));
+        exportInSpecifiedOrder.setFont(FontSize.smallFont);
 
-        ActionListener listener = new ActionListener() {
 
+        EventHandler<javafx.event.ActionEvent> listener =  new EventHandler<javafx.event.ActionEvent>() {
             @Override
-            public void actionPerformed(ActionEvent e) {
-                boolean selected = e.getSource() == exportInSpecifiedOrder;
+            public void handle(javafx.event.ActionEvent event) {
+                boolean selected = event.getSource() == exportInSpecifiedOrder;
                 exportOrderPanel.setEnabled(selected);
             }
         };
-        exportInOriginalOrder.addActionListener(listener);
-        exportInTableOrder.addActionListener(listener);
-        exportInSpecifiedOrder.addActionListener(listener);
+        exportInOriginalOrder.setOnAction(listener);
+        exportInTableOrder.setOnAction(listener);
+        exportInSpecifiedOrder.setOnAction(listener);
 
+        Label exportSortOrder = new Label(Localization.lang("Export sort order") + "  ------------------------");
+        exportSortOrder.setFont(FontSize.bigFont);
         // create GUI
-        builder.appendSeparator(Localization.lang("Export sort order"));
-        builder.append(exportInOriginalOrder, 1);
-        builder.nextLine();
-        builder.append(exportInTableOrder, 1);
-        builder.nextLine();
-        builder.append(exportInSpecifiedOrder, 1);
-        builder.nextLine();
+        builder.add(exportSortOrder, 1, 1);
+        builder.add(new Separator(), 2, 1);
+        builder.add(exportInOriginalOrder,  1, 2);
+        builder.add(new Line(), 2, 3);
+        builder.add(exportInTableOrder,  1, 4);
+        builder.add(new Line(), 2, 5);
+        builder.add(exportInSpecifiedOrder, 1, 6);
+        builder.add(new Line(), 2, 7);
 
         exportOrderPanel = new SaveOrderConfigDisplay();
```

Comment:
If you directly initialize this SaveOrderConfigDisplay Variable at declaration above, you can change your event listener to this neat lambda: 

```

      EventHandler<ActionEvent> listener = (event) -> {
            boolean selected = event.getSource() == exportInSpecifiedOrder;
            exportOrderPanel.setEnabled(selected);
        };
```

Reply:
Thanks for the advice :). Done it